examples/lading-container.yaml

```
@@ -19,4 +19,5 @@ generator:
       network_disabled: false
       exposed_ports:
         - 5000/tcp
+      max_lifetime: 20
```

```suggestion
      max_lifetime_seconds: 20
```

would prefer to be explicit so the units are clear

---

Fixed in https://github.com/DataDog/lading/pull/1325/commits/fea93299c54706d7b6aac3303ed9c80d793f00b0#diff-884b2b115b4cc2ae969b3beed17e0ea31dea56389e3948697694c795ebc7eb73R43.